API.md

```diff
@@ -360,3 +360,18 @@ angular.module('yourApp', ['validation'])
 ```
 
 
+### **Setup Message Element in config phase **
+
+```javascript
+// your module
+angular.module('yourApp', ['validation'])
+    .config(['$validationProvider', function ($validationProvider) {        
+        $validationProvider.addMsgElement = function(element) {
+          return $(element).parent().append('<span></span>');
+        };
+        $validationProvider.getMsgElement = function(element) {
+          return $(element).parent().children().last();
+        };
+
+    }]);
+```
```

Comment:
Do you mind give more description about `add/getMsgElement` ?

This is difficult to understand,

1. Show the default message element
2. Show the example why we use addMsgElement